### ISSUE TITLE
i18n'ify strings that were missed in the frontend

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#191]
### Problem

A few of the recent .vue related PRs didn't use `<i18n>` tags were they needed to and I missed this in my reviews of them (e.g. [here](https://github.com/vijayee/group-income-simple/blob/37fdb058a32f2c5229554f698523e63951c363c5/frontend/simple/views/SignUp.vue#L27), where `<span>` should be replaced with `<i18n>` -- just make sure that the span's properties are carried over to the rendered template span).

### Solution

Scan through all of the .vue files that were recently added or updated and make sure everything is using `<i18n>` or the `L` function (as appropriate).

**Comment by taoeffect** [comment]
I believe this was fixed in #195.
